stdlib/builtins.pyi

```diff
@@ -784,6 +784,9 @@ class slice:
     def indices(self, __len: SupportsIndex) -> tuple[int, int, int]: ...
 
 class tuple(Sequence[_T_co], Generic[_T_co]):
+    @overload
+    def __new__(cls) -> tuple[()]: ...
+    @overload
     def __new__(cls: type[Self], __iterable: Iterable[_T_co] = ...) -> Self: ...
```

Comment:
```suggestion
    def __new__(cls: type[Self], __iterable: Iterable[_T_co]) -> Self: ...
```
This overload should be used only if `__iterable` is provided, so there should be no default.

Reply:
Ah, because now there's 2 call signatures- one for when `__iterable` is provided, and one without. Makes sense! i'll update.